ytsclient/src/main/java/yts/request/interceptor/UserAgentInterceptor.java

```diff
@@ -0,0 +1,13 @@
+package yts.request.interceptor;
+
+import retrofit.RequestInterceptor;
+
+public class UserAgentInterceptor implements RequestInterceptor {
+
+    private static final String USER_AGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.82 Safari/537.36";
```

Comment:
I like the approach with the interceptor, but I don't think we should use a browser user agent. 
I think we should allow the client of the library to customize this. Maybe something like `withUserAgent(String)`
I will try a few things during the weekend.